packages/main/src/types/BadgeSize.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * Defines badge size types.
+ * @public
+ */
+enum BadgeSize {
+	/**
+	 * Standard size of the badge
```

Comment:
Pls. double check. "Standard" is OK, but implies that it should always be the default value.

Reply:
"S" is for small, this is old wording.